### PR TITLE
Jroverf/momentum actuator node kernel

### PR DESCRIPTION
**Pull request by overfelt**

### CODE REVIEW

File: include/node_kernels/MomentumActuatorNodeKernel.h [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/*------------------------------------------------------------------------*/
+/*  Copyright 2019 National Renewable Energy Laboratory.                  */
```
Comment (sayerhs) [inline include/node_kernels/MomentumActuatorNodeKernel.h]: @overfelt You might want to assert Sandia copyright assertion here.

File: include/node_kernels/MomentumActuatorNodeKernel.h [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+#include "node_kernels/NodeKernel.h"
+
+#include "stk_mesh/base/BulkData.hpp"
+#include "stk_ngp/Ngp.hpp"
+
+namespace sierra{
+namespace nalu{
+
+class SolutionOptions;
```
Comment (sayerhs) [inline include/node_kernels/MomentumActuatorNodeKernel.h]: Forward declaration not used?

File: include/node_kernels/MomentumActuatorNodeKernel.h (hunk 1 above)
Comment (overfelt) [reply include/node_kernels/MomentumActuatorNodeKernel.h]: Interesting.  Should we have both NREL and SNL in all files for consistency?  Or can only one of us assert copyright at a time?

File: include/node_kernels/MomentumActuatorNodeKernel.h (hunk 2 above)
Comment (overfelt) [reply include/node_kernels/MomentumActuatorNodeKernel.h]: Yes, I missed that one.

File: include/node_kernels/MomentumActuatorNodeKernel.h (hunk 1 above)
Comment (sayerhs) [reply include/node_kernels/MomentumActuatorNodeKernel.h]: @overfelt We will have to wait for Mike and Paul to weigh in on this one. I think they are working on getting a joint copyright assertion for the codebase.

**Pull request merged**